### PR TITLE
Temporarily remove Hugo Pipes Sass conversion

### PR DESCRIPTION
**Pull request by lucperkins**
Background: Netlify does not yet support the "extended" version of Hugo, which includes support for Sass conversion to CSS (which is under the [Hugo Pipes](https://gohugo.io/hugo-pipes/) feature). The current workaround for this issue is to commit the generated `resources` folder to Git. As with most instances of committing generated assets to Git, this has produced a variety of headaches. This PR removes the Hugo Pipes Sass conversion pipeline in favor of one that uses the [`sass`](https://sass-lang.com/install) CLI tool for those who need to work with Sass. Once Netlify supports the extended version of Hugo we can go back to using Hugo Pipes.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,6 +24,9 @@ production-build: build check-headers-file ## Build the production site and ensu
 non-production-build: ## Build the non-production site, which adds noindex headers to prevent indexing
 	hugo --enableGitInfo
 
+sass-build:
+	scripts/sass.sh build
```
Comment (kbhawkey) [inline Makefile]: minor nit: HUGO_VERSION is set in Makefile and netlify.toml. Is there a better place to set this variable that all scripts/configuration can use?

File: sass/_base.sass [hunk 2]
```diff
@@ -0,0 +1,1793 @@
...
+
+		&:before
+			transform: rotate(45deg)
+
+		&:after
+			transform: rotate(-45deg)
+
+.open-nav, .y-enough
+	#tryKubernetes
+		width: 150px
```
Comment (kbhawkey) [inline sass/_base.sass]: Would you accept an update here or should I continue with PR#10353?

File: sass/_base.sass [hunk 3]
```diff
@@ -996,8 +996,10 @@ dd
 	img
 		max-width: 100%
 
-	#TableOfContents > ul > li { list-style: none; }
```
Comment (kbhawkey) [inline sass/_base.sass]: Did these files get moved from assets?

File: Makefile (hunk 1 above)
Comment (lucperkins) [reply Makefile]: Unfortunately there really isn't a way around that AFAIK. Netlify needs to have that information in the config and the `Makefile` needs it for people to be able to easily build the Docker image. Ideally the `Dockerfile` could extract that information from the Netlify config but I'm not sure there's a way to do that that doesn't add a lot of complexity.

File: sass/_base.sass (hunk 2 above)
Comment (lucperkins) [reply sass/_base.sass]: I would say hold off on that. Wait for this PR to be merged, then move all of the Sass files in `assets/sass` into this root-level folder in your PR. That strikes me as the least messy option.

File: resources/OWNERS [hunk 4]
```diff
@@ -1,8 +0,0 @@
-# Allow CSS/SASS updates from blog and case studies owners
```
[the remark this answers is not in the document]
Comment (lucperkins) [reply resources/OWNERS]: Good call on that! Completely missed it. Done.

File: sass/_base.sass (hunk 2 above)
Comment (kbhawkey) [reply sass/_base.sass]: 👍

**Comment by zacharysarah** [comment]
@kbhawkey 👋 Does this PR look good enough to you to `/lgtm`?

**Pull request merged**